master/pkg/model/task.go

```diff
@@ -101,6 +101,15 @@ type Allocation struct {
 // AllocationState represents the current state of the task. Value indicates a partial ordering.
 type AllocationState int
 
+// TaskStats is the model for task stats in the database.
+type TaskStats struct {
+	AllocationID AllocationID `db:"allocation_id"`
+	ResourcePool string       `db:"resource_pool"`
+	EventType    string       `db:"event_type"`
+	StartTime    *time.Time   `db:"start_time"`
+	EndTime      *time.Time   `db:"end_time"`
```

Comment:
bun tags are `bun:` instead of `db:` so I suspect these end up doing nothing, but the field/column names luckily match anyway.

Reply:
This table is used by both bun and plain sql, I wonder if changing to `bun` may cause any problem

Reply:
if these aren't necessary here than maybe omitting them keeps the code more concise. 

but if you had to have them here, you could totally have multiple tags space-separated inside backticks, e.g.:
```
type Allocation struct {
	bun.BaseModel `bun:"table:allocations"`

	AllocationID AllocationID     `db:"allocation_id" bun:"allocation_id,notnull,unique"`
	TaskID       TaskID           `db:"task_id" bun:"task_id,notnull"`
	Slots        int              `db:"slots" bun:"slots,notnull"`
	AgentLabel   string           `db:"agent_label" bun:"agent_label,notnull"`
	ResourcePool string           `db:"resource_pool" bun:"resource_pool,notnull"`
	StartTime    *time.Time       `db:"start_time" bun:"start_time"`
	EndTime      *time.Time       `db:"end_time" bun:"end_time"`
	State        *AllocationState `db:"state" bun:"state"`
	IsReady      *bool            `db:"is_ready" bun:"is_ready"`
}
```